proxy/pom.xml

```diff
@@ -95,6 +95,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>5.3.23</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Move to dependencyManagement of parent's pom.

Reply:
Good catch